conanfile.txt

```diff
@@ -5,6 +5,7 @@ sdl2_image/2.0.5@bincrafters/stable
 sdl2_mixer/2.0.4@bincrafters/stable
 glew/2.1.0@bincrafters/stable
 glm/0.9.9.5@g-truc/stable
+libiconv/1.16@_/_
```

Comment:
```suggestion
libiconv/1.16
```

`@_/_` are just used internal and mean no username and no channel. Within `conanfile.txt` and `conanfile.py` you can and should omit this. On the command line you need to add only a `@` at the end for backwards compatibilty like `libiconv/1.16@` (this will change with Conan 2.0)

Reply:
Packages from Conan Center index do not have that https://github.com/conan-io/conan-center-index

username is unnecessary as it is maintained by the entire community and the channel is unnecessary as we don't maintain `testing` packages or similar

Reply:
OK thanks for letting me know.